sdk/core/azure-core-amqp/samples/local_client_async_sample/local_client_async_sample.cpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-Licence-Identifier: MIT
+
+#include <azure/core/amqp/connection.hpp>
+#include <azure/core/amqp/message_sender.hpp>
+#include <azure/core/amqp/models/messaging_values.hpp>
+#include <chrono>
+#include <iostream>
+#include <limits>
+#include <string>
+
+int main()
+{
+  Azure::Core::Amqp::_internal ::ConnectionOptions connectOptions;
+  connectOptions.EnableTrace = true;
+  connectOptions.ContainerId = "some";
+  Azure::Core::Amqp::_internal::Connection connection("amqp://localhost:5672", connectOptions);
```

Comment:
Why this port?

Reply:
[See](https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=AMQP)

Reply:
It might be useful to link to that doc as a comment, for the benefit of any consumer without domain knowledge who is wondering what this means.